client/views/home.ejs

```diff
@@ -1,145 +1,145 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-180152810-1"></script>
-    <script>
-      window.dataLayer = window.dataLayer || [];
 
-      function gtag() {
-        dataLayer.push(arguments);
-      }
-      gtag('js', new Date());
+<head>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-180152810-1"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
 
-      gtag('config', 'UA-180152810-1');
-    </script>
-    <title>Holiday Gift Donation | Donate Gifts to Foster Kids -Virtual Toy Drive</title>
-    <!-- ** NOTE: CDN Bootstrap.js, Popper.js, and jQuery are moved to the bottom so the page loads faster-->
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag('js', new Date());
 
-    <!-- Site SEO -->
-    <meta
-      name="description"
-      content="How to help children in foster care | Send holiday birthday gifts to foster kids easily by using embedded Amazon delivery"
-    />
-    <meta name="”robots”" content="index, follow" />
-    <meta name="keywords" content="children, gift, wish, foster, donate" />
-    <%- include('./templates/header')-%>
-  </head>
+    gtag('config', 'UA-180152810-1');
+  </script>
+  <title>Holiday Gift Donation | Donate Gifts to Foster Kids -Virtual Toy Drive</title>
+  <!-- ** NOTE: CDN Bootstrap.js, Popper.js, and jQuery are moved to the bottom so the page loads faster-->
 
-  <body>
-    <div id="fb-root"></div>
-    <script
-      async
-      defer
-      crossorigin="anonymous"
-      src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v8.0"
-      nonce="XBZR4Yw2"
-    ></script>
-    <%- include('./templates/preloader')-%> <%- include('./templates/nav')-%> <%-
+  <!-- Site SEO -->
+  <meta name="description"
+    content="How to help children in foster care | Send holiday birthday gifts to foster kids easily by using embedded Amazon delivery" />
+  <meta name="”robots”" content="index, follow" />
```

Comment:
""robots"" with double the amount of quotes?

Reply:
great eyes! typo